build_hangover_arm

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Checkout hangover repository
+if [ ! -d hangover ]; then
+    git clone  --depth=1 git@github.com:AndreRH/hangover.git
+    cd hangover
+    git submodule update --init --depth=1
+    cd ..
+else
+    cd hangover
+    git fetch && git pull origin master
+    git submodule update --depth=1
+    cd ..
+fi
+
+# Build hangover
+docker build \
+    -f hangover/Dockerfile.cross_arm64 \
+    -t "hangover-builder" \
+    hangover
+
+# Extract the build files from the hangover-builder image
+docker run --rm hangover-builder tar -C /root/hangover/ --exclude-vcs -czf - . > hangover.tar.gz
+
+docker build \
+    --platform="linux/arm64" \
```

Comment:
can you do platform on docker build? i thought you had to do docker buildx build --platform

Reply:
Yep, Buildkit is the default build engine on Docker Desktop for Mac (and docker-ce for Linux and I assume in other versions too), so you can drop the buildx commands and don’t even need to use —load or buildx create.